src/eei.h

```diff
@@ -33,15 +33,15 @@ struct ExecutionResult {
   bool isRevert = false;
 };
 
-struct EthereumInterface : ShellExternalInterface {
+struct EthereumInterface {
```

Comment:
Make it `class`.

Reply:
Yeah not sure why it was still a struct.